dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -49,13 +65,21 @@ fi
 
 if [ -z "$CONFIG_TYPE" ] || [ "$CONFIG_TYPE" == "separated" ]; then
     echo "no service integrated-vtysh-config" > /etc/frr/vtysh.conf
+    sonic-cfggen $CFGGEN_PARAMS_SEPARATED
     rm -f /etc/frr/frr.conf
 elif [ "$CONFIG_TYPE" == "unified" ]; then
     echo "service integrated-vtysh-config" > /etc/frr/vtysh.conf
+    sonic-cfggen $CFGGEN_PARAMS_UNIFIED
+    rm -f /etc/frr/bgpd.conf /etc/frr/zebra.conf /etc/frr/staticd.conf
+elif [ "$CONFIG_TYPE" == "split" ]; then
+    echo "service integrated-vtysh-config" > /etc/frr/vtysh.conf
+    sonic-cfggen $CFGGEN_PARAMS_SPLIT
```

Comment:
unrelated changes. can you pull to a different pr?

Reply:
ok